libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -326,8 +326,8 @@ void AP_AHRS_NavEKF::update_SITL(void)
             // rotate earth velocity into body frame and calculate delta position
             Matrix3f Tbn;
             Tbn.from_euler(radians(fdm.rollDeg),radians(fdm.pitchDeg),radians(fdm.yawDeg));
-            Vector3f earth_vel = Vector3f(fdm.speedN,fdm.speedE,fdm.speedD);
-            Vector3f delPos = Tbn.transposed() * (earth_vel * delTime);
+            const Vector3f earth_vel = Vector3f(fdm.speedN,fdm.speedE,fdm.speedD);
```

Comment:
Why no {} here ?

Reply:
because it leads to a narrowing conversion from double to float .

Reply:
You can call the constructor though.

Reply:
nice, I haven't though about that :-S . Done